ot/lp/__init__.py

```diff
@@ -443,18 +443,30 @@ def free_support_barycenter(measures_locations, measures_weights, X_init, b=None
     - we do not optimize over the weights
     - we do not do line search for the locations updates, we use i.e. theta = 1 in [1] (Algorithm 2). This can be seen as a discrete implementation of the fixed-point algorithm of [2] proposed in the continuous setting.
 
+    Formally we solve the following problem:
```

Comment:
Could you please put the optimization problem after the first sentence and before the discussion on the algorithm?

Also the itemize in the discussion does not work (it needs an empty line maybe)